### PR TITLE
Board member integration test

### LINKED ISSUE #233: Board Member Integration Tests
# Description
Write integration tests for the board member tag

# Acceptance Criteria
- [ ] Tests cover all cases that cross from the API layer to the DB layer
- [ ] Unit tests are updated
- [ ] Unit tests pass

### PR DESCRIPTION
**Pull request by jjirinec**
closes #233

### CODE REVIEW

File: src/__tests__/database/server/boardMember.test.ts [hunk 1]
```diff
@@ -119,24 +119,23 @@ describe('patchBoardMember()', () => {
     it('Throws for user error', async () => {
         const boardId = 1451;
         await expect(patchBoardMember(boardId, { })).rejects.toThrow('user input error');
-        expect(mockQuery).toHaveBeenCalled();
```
Comment (matedro) [inline src/__tests__/database/server/boardMember.test.ts:122]: This test was supposed to cover the 1451 case in the error handling, so I'd suggest adding a request body instead of removing the mock query assertion. There should also be another test that's testing for the empty request like you're doing here.

File: src/__tests__/integration/boardMember.test.ts [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+        expect(res.status).toBe(201);
+        const boardMember: BoardMember = res.body;
+        expect(boardMember.year).toBe(2023);
+        expect(boardMember.memberId).toBe(1);
+        expect(boardMember.title).toBe('President');
+    });
+});
+
+describe('GET /boardMember/:boardMemberId', () => {
+    it.skip('Returns 401 for no token', async () => {
```
Comment (matedro) [inline src/__tests__/integration/boardMember.test.ts:104]: Why is this skipped?

**Review by matedro — CHANGES_REQUESTED** [review]

File: src/__tests__/database/server/boardMember.test.ts (hunk 1 above)
Comment (jjirinec) [reply src/__tests__/database/server/boardMember.test.ts:122]: Good catch, In the new commit I added a test for the 1451 case in `patchBoardMember.`

I also added a test to catch the empty request user error in `insertBoardMember` as well

**Review by matedro — APPROVED** [review]
